instrumentation/src/main.c

```diff
@@ -0,0 +1,101 @@
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <stdio.h>
+
+#define MAX_LINE_LENGTH 1023
+#define MAX_LINES 50
+
+// TODO change to systemd drop in file paths
+static char *const env_var_file_java = "/etc/splunk/zeroconfig_java.conf";
+static char *const env_var_file_node = "/etc/splunk/zeroconfig_node.conf";
```

Comment:
@jeffreyc-splunk would you have the path of the systemd drop-in env var files handy? Did you also separate them by language by any chance?

Reply:
If we go with https://github.com/signalfx/splunk-otel-collector/pull/3506, the deb/rpm packages can include sample systemd drop-in files in `/usr/lib/splunk-instrumentation/examples/systemd/` for each language (see https://github.com/signalfx/splunk-otel-collector/pull/3506/files#diff-71d510eec30ef5d3556193068fc6726c24202c439f5bd9f25989cb511ff113f9 for java). We can then add a sample `splunk-otel-js.conf` once we add support for the node.js agent.  These sample files can be used for manual installations.

On the other hand, I'm thinking that the installers will automatically create the actual drop-in file to be loaded by systemd, for example `/usr/lib/systemd/system.conf.d/splunk-otel-auto-instrumentation.conf`, based on the installation options and defaults (see https://github.com/signalfx/splunk-otel-collector/pull/3506/files#diff-4dca014cfa3f56017a31c6a9e698de416a5bcc9cf6450a8ef9d088295f19ca98 for the test config).

Reply:
Side note: A reoccurring issue we have been noticing with being swappable with upstream auto-instrumentation solutions vs Splunk solutions is Splunk file paths are unique to Splunk vs what upstream uses. I don't think this issue should be addressed in this PR, but it is worth noting for future compatibility.

Reply:
We can make those configurable via macros. I'll do that.

Reply:
Done.